src/freedv_api.c

```diff
@@ -340,9 +340,20 @@ void freedv_tx(struct freedv *f, short mod_out[], short speech_in[]) {
         }
         freedv_tx_fsk_voice(f, mod_out);
     } else {
+        // Temporarily enable real-only BPF to improve performance.
+        if (FDV_MODE_ACTIVE( FREEDV_MODE_700D, f->mode) || FDV_MODE_ACTIVE( FREEDV_MODE_700E, f->mode)) {
+            ofdm_set_realonlybpf(f->ofdm, 1);
```

Comment:
Any particular reason why we only run this for 700D & E?

Reply:
Those were the only OFDM modes that came to mind offhand, actually. We can add additional ones if there's a need but given that those are the only two we can run on the embedded platforms at the moment, it probably wouldn't hurt to keep as is?